grammars/ini.cson

```diff
@@ -8,6 +9,14 @@
 'name': 'INI'
 'patterns': [
   {
+    'match': '\\s(\\d+)\\s'
```

Comment:
Thanks to `ini` playing it fast and loose, these values won't always be surrounded by whitespace. Both `example=123` and `example = 123` are valid.